app/models/user_target.rb

```diff
@@ -0,0 +1,4 @@
+class UserTarget < ApplicationRecord
+  belongs_to :user, dependent: :destroy
```

Comment:
Are you sure that you want to destroy the user associated when the UserTarget is deleted?
Check: https://apidock.com/rails/v4.2.7/ActiveRecord/Associations/ClassMethods/belongs_to